tests/test_workers/test_tasks/test_build.py

```diff
@@ -260,6 +260,29 @@ def test_push_image(mock_si, mock_run_cmd, mock_glps, schema_version):
         mock_run_cmd.assert_called_once()
 
 
+@mock.patch('iib.workers.tasks.build.run_cmd')
+def test_fix_schema_version_fail_retry_success(mock_run_cmd):
```

Comment:
My preference is to test the retry decorator on its own rather than test individual functions that use the retry decorator. The reason is that once the `_fix_schema_version` function is removed, this test will get deleted along with it. It also makes the tests more independent.

For example (untested):
```python
def test_retry():
    mock_func = mock.Mock()

    @retry(retries=3, wait_on=IIBError)
    def _func_to_retry():
        mock_func()
        raise IIBError('Some error')

    with pytest.raises(IIBError, match='Some error'):
        _func_to_retry()

    assert mock_func.call_count == 4
```